docs/intro.md

```diff
@@ -1,7 +1,8 @@
 ---
 id: intro
-slug: /
```

Comment:
I think slug is needed here since the intro page is the root page.
If we remove slug here, we can use https://docusaurus.io/docs/docusaurus-core#redirect to redirect to into each time user hit the root `/`.

Reply:
Thank you so much for your review @Kevan-Y ! I reinserted `slug: /` into the intro.md and added an extra step in the Workflow in Contributing.md. Could you please review them? If I need to improve, please let me know!